frontend/pages/app/sign-in.vue

```diff
@@ -1,9 +1,36 @@
 <script lang="ts" setup>
+import { ref } from 'vue';
+
+
+const email = ref('');
+const password = ref('');
 definePageMeta({
   layout: "auth-layout",
 });
-const signIn = () => {
-  useRouter().push('/app')
+const signIn = async () => {
+  try {
+    const response = await fetch('http://localhost:9001/api/v1/auth', {
+      method: 'POST',
+      mode: 'no-cors',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        email: email.value,
+        password: password.value
+      })
+    });
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+
+    const data = await response.json();
+    document.cookie = `jwt=${data.response_data.jwt}; path=/`;
```

Comment:
Don't set the cookie here. It will be an HTTP-only cookie. Instead, just check that the response is correct (200) and then forward them. If it is incorrect, pop up an error message appropriately.